configure.py

```diff
@@ -777,6 +777,10 @@ def check_compiler(o):
     # to a version that is not completely ancient.
     warn('C compiler too old, need gcc 4.2 or clang 3.2 (CC=%s)' % CC)
 
+  # Fix based on work of @bnoordhuis from 
+  # https://github.com/nodejs/node-v0.x-archive/issues/4186 .
+  o['variables']['clang'] = 1 if is_clang else 0
+  o['variables']['gcc_version'] = gcc_version[0] * 10 + gcc_version[1]
```

Comment:
This isn't used anymore so should be removed.

Reply:
@richardlau We still need 'clang' to enable `-Wno-old-style-declaration` for linux.
https://github.com/nodejs/node/blob/4d792bbe807d30b7c8f3af379f27f0747958faa2/deps/openssl/openssl_common.gypi#L66-L69
since clang doesn't seem to support this flag.
I couldn't find this `clang` variable being set anywhere in the configure.py .

Reply:
I was referring to `gcc_version`.

Reply:
Done.
I found out that we can also use `llvm_version` instead of clang for the check.